app/src/main/resources/openapi_etor.yaml

```diff
@@ -40,3 +52,14 @@ components:
         patientId:
           type: string
           example: MRN7465737865
+    UnauthorizedError:
+      description: Unauthorized Request
+    InternalError:
+      description: Internal Server Error
+  securitySchemes:
+    ApiKeyAuth:
+      type: apiKey
+      in: header
+      name: OUR-API-KEY
+security:
+  - ApiKeyAuth: []
```

Comment:
I wouldn't apply this auth to everything.  Let's apply it on the operation level.  I say this because while everything in this file, thus far, will need to be protected, this file will be combined with other plugin's OpenAPI specification which may have endpoints that don't use this API key auth.